plugins/env/app/controllers/environment_variables_controller.rb

```diff
@@ -10,8 +11,29 @@ def index
     @pagy, @environment_variables = pagy(scope, page: params[:page], items: 30)
   end
 
+  def show
+    render 'form'
+  end
+
+  def update
+    environment_variable.attributes = attributes
+    environment_variable.save!
```

Comment:
can combine these 2 lines to `environment_variable.update_attributes!(attributes)`